ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.Data/Repositories/TaskRepository.cs

```diff
@@ -48,5 +48,26 @@ public Task BulkCreateAsync(List<Models.Task> tasks, CancellationToken cancellat
             cancellationToken.ThrowIfCancellationRequested();
             return GetCollection<Models.Task>().InsertManyAsync(tasks);
         }
+
+        public Task<List<Models.Task>> EmailApproverAsync(List<string> approverNames, List<string> approverEmails, string employeeName, Guid employeeId, CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+
+            var filterBuilder = Builders<Models.Task>.Filter;
+            var employeeFilter = filterBuilder.Eq(t => t.EmployeeId, employeeId);
+            var statusFilter = filterBuilder.Eq(t => t.Status, Models.Enums.Status.WaitingForApproval);
+
+            return GetCollection<Models.Task>().Find(/*statusFilter &*/ employeeFilter).SortBy(t => t.ApproverId).ToListAsync();
+        }
+
+        public Task<List<Guid>> GetApproversAsync(Guid id, CancellationToken cancellationToken)
```

Comment:
it's not clear from the name of argument which id do you mean. Usually if we write id in TaskRepository it's considered to be Task id

Reply:
rename argument both in class and interface